js-style.xml

```diff
@@ -16,6 +16,7 @@
     <option name="SPACES_WITHIN_OBJECT_LITERAL_BRACES" value="true" />
     <option name="SPACES_WITHIN_IMPORTS" value="true" />
     <option name="SPACES_WITHIN_INTERPOLATION_EXPRESSIONS" value="true" />
+    <option name="IMPORT_SORT_MODULE_NAME" value="true" />
```

Comment:
I feel like there was some weird result when I tried this before, but that may have been before we were using full paths for our modules.